modules/search/instant-search/components/search-filter-dates.jsx

```diff
@@ -0,0 +1,42 @@
+/** @jsx h */
+
+/**
+ * External dependencies
+ */
+import { h, Component } from 'preact';
+import strip from 'strip';
+
+export default class SearchFilterDates extends Component {
+	render() {
+		return (
+			<div>
+				<h4 className="jetpack-search-filters-widget__sub-heading">{ this.props.filter.name }</h4>
+				<ul className="jetpack-search-filters-widget__filter-list">
+					{ this.props.aggregation &&
+						'buckets' in this.props.aggregation &&
+						[
+							...this.props.aggregation.buckets
+								// TODO: Remove this filter; API should only be sending buckets with document counts.
+								.filter( bucket => !! bucket && bucket.doc_count > 0 )
+								.map( bucket => (
+									<div>
+										<input
+											disabled
+											id={ `jp-instant-search-filter-dates-${ bucket.key_as_string }` }
+											name={ bucket.key }
+											type="checkbox"
+										/>
+										<label htmlFor={ `jp-instant-search-filter-dates-${ bucket.key_as_string }` }>
+											{ strip( bucket.key_as_string ) } ({ bucket.doc_count })
```

Comment:
Could we push the interval formatting and localization to the API? Otherwise, we'll need to consider adding a date formatting library to the client, which could add a minimum of 15kB to our production bundle.

The interval value is already being passed in the [API call](https://github.com/Automattic/jetpack/pull/13332/files#diff-f51cdad93030f6c9c3c7917865a7e7afR26).

Reply:
Possibly ya. Is the 15kb from pulling in https://www.npmjs.com/package/@wordpress/date which looks like it includes date_i18n()? We may want something like this anyways to correctly display in search results though.